scripts/build_funcs.sh

```diff
@@ -184,6 +184,8 @@ native() {
     LDFLAGS="-s -w -X $GOTARGET/pkg/buildinfo.Version=$GIT_VERSION -X $GOTARGET/pkg/buildinfo.GitSHA=$GIT_REF_LONG"
     args=(-ldflags "${LDFLAGS}" "$GOTARGET")
     CGO_ENABLED=0 GOOS="$HOST_GOOS" GOARCH="$HOST_GOARCH" go build -o sonobuoy "${args[@]}"
+    mkdir -p ./build/$HOST_GOOS/$HOST_GOARCH
+    cp ./sonobuoy ./build/$HOST_GOOS/$HOST_GOARCH/sonobuoy
```

Comment:
Just incidental improvement, not really related but doesnt deserve its own PR to improve a small local method.